src/core/components/forms/Select.tsx

```diff
@@ -91,12 +91,12 @@ function Select<O>(props: SelectProps<O>) {
       loading={loading}
       withPortal
     >
-      {onCreate && query.length > 0 && (
+      {onAdd && query.length > 0 && (
         <div
           className="cursor-pointer p-2 text-gray-900 hover:bg-blue-500 hover:text-white"
-          onClick={handleCreate}
+          onClick={handleAdd}
         >
-          Create &quot;{query}&quot;
+          Add &quot;{query}&quot;
```

Comment:
Since it's a shared component, it would be better to take the prefix "Add..." from the props.